### DISCUSSION TITLE
Light Mode Selected on Entity Card  should be sent to backend

### DISCUSSION THREAD

**Discussion by icemanch** [home-assistant/frontend#10210] [Other feature requests] [answered]
I"m not sure if this is a bug or a feature request but, I'll start here.

The front end currently offers the ability to pick between color or temperature when using a bulb that supports RGB and CCT.  This is great as it gives the option to use color or temperature when changing the apperance of a bulb.

However, there is a problem when adjusting the brightness.  Currently, there are four sources of brightness information that are used on the light card when RGBWW is supported.  

RGB -> Color Brightness 
W  -> Cold White Brightness
W -> Warm White Brightness  

Brightness - Brightness (The average of all the channels)

![image](https://user-images.githubusercontent.com/10890432/136584163-c7826d87-3d52-43c8-9820-291f9f83a766.png)

While in color mode adjusting the main brightness should adjust each channel brightness (depends on the integration).

When switching to temperature there is nothing being sent back to the integration to let it know the user has selected temperature.  So, the brightness is still tied to the average of the channels since the integration wasn't told of the change. The integration could recalculate the brightness once it receives a CCT but, until that happens the brightness will not function correctly.  

I really think that the frontend should either send a different brightness (CCT_BRIGHTNESS) or should tell the integration that the user has selected a different mode.

**Comment by bramkragten** [comment, marked as answer]
The frontend can't send a color temperature before you have picked one, so the color mode in the backed changes as soon as you select a color temperature.
